packages/teleterm/src/ui/services/workspacesService/workspacesService.ts

```diff
@@ -289,22 +279,6 @@ export class WorkspacesService extends ImmutableStore<WorkspacesState> {
     }
   }
 
-  private removeExpiredAssumedRoles(
-    assumed: Record<string, AccessRequest>,
-    user: LoggedInUser
-  ) {
-    const validRequests = {};
-    const requests = Object.values(assumed);
-    requests.forEach(request => {
-      const expired = isPast(new Date(request.expires));
-      // only add any assumed that still exist on the loggedInUser cert
-      if (user.activeRequestsList.includes(request.id) && !expired) {
-        validRequests[request.id] = request;
-      }
-    });
-    return validRequests;
-  }
-
```

Comment:
Can I safely remove it? It seems to me that `tshd` shouldn't return expired requests.